docs/04-guides/listen-for-webhooks.mdx

```diff
@@ -31,7 +31,16 @@ to a URL that you specify.
 | recording.waiting        | This fires after a stream with recording on has concluded and is not yet ready to be downloaded. Typically it takes 5 minutes for recordings to be ready for download. |
 | multistream.connected    | This fires when we've successfully connected to the multistream target                                                                                                 |
 | multistream.error        | This fires when we've encountered an error either while attempting to connect to the third party streaming service or while broadcasting.                              |
-| multistream.disconnected | This fires when we are no longer sending video to the multistream target.                                                                                              |
+| multistream.disconnected | This fires when we are no longer sending video to the multistream target.                                                                                             
```

Comment:
I think this line is missing a pipe (`|`) at the end! Also line `38` and line `40`. Doesn't seem to affect how it's viewed tho?

Reply:
@gioelecerati 
That is strange that it would still show, added it in the commit below ⬇️